src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Confirm.tsx

```diff
@@ -481,8 +519,8 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
 
   const canWithdraw = useMemo(() => {
     const amountCryptoBaseUnit = toBaseUnit(state?.withdraw.cryptoAmount, asset.precision)
-    return bnOrZero(amountCryptoBaseUnit).gte(protocolFeeCryptoBaseUnit)
-  }, [state?.withdraw.cryptoAmount, asset.precision, protocolFeeCryptoBaseUnit])
+    return bnOrZero(amountCryptoBaseUnit).gte(isRunePool ? 0 : protocolFeeCryptoBaseUnit)
```

Comment:
Confirmed there are no protocol fees involved for deposits - not sure yet about withdraws as they're not going through, but not a now thing anyway.

Can we add a TODO somewhere to be cognicent of that when the first deposits reach withdraw maturity? 

i.e .9 RUNE deposited

<img width="278" alt="image" src="https://github.com/user-attachments/assets/42ea30d8-f71f-47d4-b33f-36f80628d2ca">
<img width="457" alt="image" src="https://github.com/user-attachments/assets/2853f2e3-6b1c-4506-8980-48d91543dfc0">